db/patch.js

```diff
@@ -4,4 +4,4 @@
 
 // The expected patch level of the database. Update if you add a new
 // patch in the schema/ directory.
-module.exports.level = 7
```

Comment:
We can chat about this later today, but this will cause problems for fxa-dev instances that will have already patched up to revision 7. Bleh.